carte/static/games/scopa.css

```diff
@@ -0,0 +1,245 @@
+          (
+            round(down, var(--card-field-position) / var(--playing-area-wrap)) -
+            round(down, (var(--card-field-size) - 0.5) / var(--playing-area-wrap)) /
+            2
+          ) *
+          (var(--gap) + var(--card-height))
+        );
+        left: calc(
+          var(--center-x) +
+          (var(--row-position) - (var(--row-size) - 1) / 2) *
+          (var(--gap) + var(--card-width))
+        );
+      }
+
+      &[data-position="hand"] {
+        --active-card-offset: 0;
+        left: calc(
+          var(--center-x) +
+          (var(--card-field-position) - (var(--card-field-size) - 1) / 2) *
+          (var(--gap) + var(--card-width))
+        );
+        &[data-player="opponent"] {
+          top: calc(var(--top) + var(--active-card-offset) * var(--card-height));
+        }
+        &[data-player="self"] {
+          top: calc(var(--bottom) - var(--active-card-offset) * var(--card-height));
+        }
+      }
+
+      &[data-position="points"] {
+        --card-rotation: 1;
+        --deck-writing-mode: vertical-rl;
+        /* z-index of 18 to be above of all the scopa-points card (at most 18) */
+        z-index: 18;
+        left: calc(var(--left) + var(--rotation-diff));
+        &[data-player="opponent"] {
+          top: calc(var(--top) - var(--rotation-diff));
+        }
+        &[data-player="self"] {
+          top: calc(var(--bottom) - var(--deck-shadow-width) + var(--rotation-diff));
+        }
+      }
+
+      &[data-position="points-scopa"] {
+        --num-offset: calc(
+          2 *
+          var(--card-field-position) /
+          (var(--card-field-size) - 1) -
+          1
+        );
+        &[data-field-size="1"] {
+          --num-offset: 0;
+        }
+        left: calc(
+          /* base distance from the left: center */
+          var(--left) +
+          var(--deck-shadow-width) +
+          var(--rotation-diff) +
+          /* card based count */
+          (var(--rotation-diff) - var(--card-radius) / 2) *
+          var(--num-offset)
+        );
+        z-index: var(--card-field-position);
+        &[data-player="opponent"] {
+          top: calc(var(--top) + var(--deck-shadow-width));
+          --card-rotation: 2; /* upside down */
+        }
+        &[data-player="self"] {
+          top: calc(var(--bottom));
+        }
+      }
+    }
+
+    &.name-tag {
+      left: var(--gap);
+      &[data-player="opponent"] {
+        top: calc(var(--gap) * 2 + var(--card-height) + var(--deck-shadow-width));
+      }
+      &[data-player="self"] {
+        bottom: calc(var(--gap) * 2 + var(--card-height) + var(--deck-shadow-width));
+      }
+    }
+  }
+}
+
+#results {
+  #results-table {
+    tr > td[data-details] {
+      padding-top: 0.5em;
+      padding-bottom: 0.5em;
+
+      > div[data-detail-type] {
+        font-weight: normal;
+        font-size: 0.9em;
+        line-height: 1.5em;
+
+        color: var(--background-primary-highlight-strong);
```

Comment:
Please create a new variable in main.css, something like --text-color-secondary)

Reply:
ok